### PR TITLE
added maintiners in place of project admin

### LINKED ISSUE #805: Remove the Project Admin Portion from README
## Description 

Remove the Project Admin Portion from the README file. As of now, my picture and details are displayed on the README as the Project Admin. 

As the Project has grown, it would be better if the term "Project Admin" is replaced by "Maintainers". As of now, these are the official maintainers of the Project: 

- Harsh Mishra ( @harshcasper ) 
- Soham Shah ( @sohamsshah ) 
- Seema Saharan ( @seema1711 )
- Vybhav Chaturvedi ( @vybhav72954 )

Create a new section in the README by replacing the existing one, and add all their details in a similar format that exists right now.

### PR DESCRIPTION
**Pull request by haripriya9647**
As this project is maintained by many folks soo projects admin is replaced by project maintainers.

Fixes #805 

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [ ] Readme Update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,13 +141,30 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+
+|                                     <a href="https://github.com/vybhav72954"><img src="https://avatars.githubusercontent.com/u/49750343?s=400&u=191d8b88762720b0d229531fb1118e9715f0489b&v=4" width=150px height=150px /></a>                                      |
+| :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|                                                                                      **[Vybhav Chaturvedi](https://www.linkedin.com/in/vybhav-chaturvedi-0ba82614a/)**                                                                                       |
+| <a href="https://www.linkedin.com/in/vybhav-chaturvedi-0ba82614a/"><img src="https://mpng.subpng.com/20180324/vhe/kisspng-linkedin-computer-icons-logo-social-networking-ser-facebook-5ab6ebfe5f5397.2333748215219374063905.jpg" width="32px" height="32px"></a> |
```
Comment (HarshCasper) [inline README.md:164]: They look stacked vertically. Can you make them look stacked horizontally for better readability?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
Hi there!!!

Even after your latest commit, I think they are stacked to the top to bottom

**Comment by vybhav72954** [comment]
I am pretty sure you can sort this out by using `<table>` tag

For more info, click [here](https://www.w3schools.com/html/html_tables.asp)

**Review by vybhav72954 — APPROVED** [review]
LGTM

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Make an atomic commit by pushing all your commits into one single atomic commit. The overall `diff` is very small, so we don't need 5 commits for the same

**Pull request merged**